config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml

```diff
@@ -125,3 +125,126 @@ periodics:
         requests:
           cpu: 2
           memory: "6Gi"
+
+- name: ci-kubernetes-e2e-gci-gce-scalability-watch-list-off
+  cluster: k8s-infra-prow-build
+  interval: 24h
+  tags:
+    - "perfDashPrefix: watch-list-off"
+    - "perfDashJobType: watchlist-off"
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-e2e-scalability-common: "true"
+    preset-e2e-scalability-periodics: "true"
+    preset-e2e-scalability-periodics-master: "true"
+  decorate: true
+  decoration_config:
+    timeout: 120m
+  extra_refs:
+    - org: kubernetes
+      repo: kubernetes
+      base_ref: master
+      path_alias: k8s.io/kubernetes
+    - org: kubernetes
+      repo: perf-tests
+      base_ref: master
+      path_alias: k8s.io/perf-tests
+  annotations:
+    testgrid-dashboards: sig-scalability-experiments
+    testgrid-tab-name: watchlist-off
+  spec:
+    containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230406-23cb1879e3-master
+        command:
+          - runner.sh
+          - /workspace/scenarios/kubernetes_e2e.py
+        args:
+          - --check-leaked-resources
+          - --env=CONTAINER_IMAGE=registry-sandbox.k8s.io/pause:3.1 #TODO(ameukam): revert when registry.k8s.io is ready
+          - --extract=ci/latest
+          - --gcp-node-image=gci
+          - --gcp-master-size=n1-standard-4
```

Comment:
@wojtek-t wants bigger machines for both nodes.

we could use `n1-standard-8` for the master node and `n1-standard-4` or even `n1-standard-8` for the worker node

Reply:
We're validating performance of control-plane, so let's really ensure that we are not doing performance qualification of the VMs.

let's even use e2-standard-32 for master, and let's use e2-standard-8 for node for now.

Reply:
okay, we can always decrease/increase in the future.